config/environments/test.rb

```diff
@@ -7,7 +7,7 @@
 
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
-  config.cache_classes = true
+  config.cache_classes = false
```

Comment:
Could this slow the tests down when running the whole suite?

Reply:
They are marginally slower. I'll have a think.